### PR TITLE
Fix the kube-proxy document

### PR DESCRIPTION
**Pull request by 22dm**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Fix the kube-proxy comment so that the document can be generated correctly. The document of ProxyMode struct is missing some text now: https://pkg.go.dev/k8s.io/kube-proxy@v0.19.3/config/v1alpha1#ProxyMode

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by 22dm** [comment]
CLA done

**Review by neolit123 — COMMENTED** [review]
/ok-to-test
/priority backlog

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

File: staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go [hunk 1]
```diff
@@ -166,6 +166,8 @@ type KubeProxyConfiguration struct {
 	DetectLocalMode LocalMode `json:"detectLocalMode"`
 }
 
+// ProxyMode represents modes used by the Kubernetes proxy server.
+//
```
Comment (oomichi) [inline staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go:170]: This change fixes a golint failure and this PR needs to contain removal of the line of `staging/src/k8s.io/kube-proxy/config/v1alpha1` from hack/.golint_failures file as pull-kubernetes-verify detail.
```
Some packages in hack/.golint_failures are passing golint. Please remove them.

  staging/src/k8s.io/kube-proxy/config/v1alpha1
```

**Review by oomichi — COMMENTED** [review]
/cc @oomichi

File: staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go (hunk 1 above)
Comment (22dm) [reply staging/src/k8s.io/kube-proxy/config/v1alpha1/types.go:170]: done, thanks

**Review by oomichi — COMMENTED** [review]
Thanks for updating.

/lgtm

**Pull request merged**